onnx/test/shape_inference_test.py

```diff
@@ -619,70 +702,116 @@ def test_reshape_dynamic_shape_known_rank(self, _, version) -> None:
 
     @parameterized.expand(all_versions_for("Reshape"))
     def test_reshape_dynamic_shape_symbolic(self, _, version) -> None:
+        seed_values: list[tuple[str, TensorProto.DataType, Any]] = [
```

Comment:
A seed tends to make readers expect random generation? Would « base » be more accurate?